packages/@sanity/base/src/components/icons/CircleCheck.js

```diff
@@ -1,3 +0,0 @@
-import CircleCheck from 'react-icons/lib/fa/check-circle'
```

Comment:
Is this removed intentionally?

Reply:
Yes, and replaced with MdCheckCirle. But just found a place using the old one.

Reply:
Ok. Just make sure the part-name stays the same since it can be used in a plugin or some others component.